ci-operator/step-registry/qe-rhel-jetson/pytest/qe-rhel-jetson-pytest-commands.sh

```
@@ -0,0 +1,58 @@
+chmod 600 "${SSH_KEY}"
+
+EFFECTIVE_HOST="${JETSON_HOSTNAME}"
+EFFECTIVE_PORT=22
+
+# If a jumphost is configured, open a local SSH tunnel through it
+if [[ -n "${JUMPHOST:-}" ]]; then
+    LOCAL_PORT=2222
+    echo "=== Setting up SSH tunnel via jumphost ${JUMPHOST} ==="
+    ssh -o StrictHostKeyChecking=accept-new \
+        -o ExitOnForwardFailure=yes \
+        -i ${SSH_KEY} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)